tests/src/tools/ReadyToRun.SuperIlc/CompileDirectoryCommand.cs

```diff
@@ -47,11 +50,12 @@ public static int CompileDirectory(DirectoryInfo toolDirectory, DirectoryInfo in
                 runner = new CrossgenRunner(toolDirectory.ToString(), inputDirectory.ToString(), outputDirectory.ToString(), referencePath?.Select(x => x.ToString())?.ToList());
             }
 
-            if (outputDirectory.Exists)
+            string runnerOutputPath = runner.GetOutputPath();
+            if (Directory.Exists(runnerOutputPath))
             {
                 try
                 {
-                    outputDirectory.Delete(recursive: true);
+                    Directory.Delete(runnerOutputPath, recursive: true);
```

Comment:
Nit: Could you update the error message in the catch to refer to `runnerOutputPath` instead of the now potentially incorrect `outputDirectory`?

Reply:
Good point, will do!